weaver/tests/network-setups/fabric/dev/scripts/setupCC.sh

```diff
@@ -1,11 +1,12 @@
 directory=$(dirname $0)
 
+CACTI_VERSION=v2.0.0-alpha.1
 TMP_PATH=$PWD/../shared/tmp
 CHAINCODE_PATH=$PWD/../shared/chaincode
 rm -rf $CHAINCODE_PATH/interop
 
 # interop cc module
-INTEROPCC_MOD=github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/contracts/interop
+INTEROPCC_MOD=github.com/hyperledger/cacti/weaver/core/network/fabric-interop-cc/contracts/interop/v2
```

Comment:
instead of hardcoding `v2` take it from first number of `CACTI_VERSION` variable.

Reply:
Fixed